conformity/resource_conformity_communication_setting.go

```diff
@@ -419,6 +424,10 @@ func resourceConformityCommSettingCreate(ctx context.Context, d *schema.Resource
 	payload.Data.Attributes.Type = "communication"
 	payload.Data.Attributes.Enabled = d.Get("enabled").(bool)
 
+	if channel == "sns" || channel == "service_now" {
```

Comment:
Do we need to validate this here? If it doesn't cause harm, it's probably best not to validate so this is future-proof for when other channels start supporting the manual flag.